src/base58.c

```diff
@@ -26,50 +26,50 @@ static const uint32_t MAX_BUFFER_SIZE = 124;
 static const char BASE58ALPHABET[] = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
 
 size_t encode_base58(
-        const uint8_t *in, size_t length,
-        uint8_t *out, size_t maxoutlen
+        const uint8_t* inBuffer, size_t inSize,
+        uint8_t* outBuffer, size_t maxOutSize
 )
 {
-	uint8_t tmp[MAX_BUFFER_SIZE];
+	uint8_t tmpBuffer[MAX_BUFFER_SIZE];
 	uint8_t buffer[MAX_BUFFER_SIZE * 2];
```

Comment:
I am not thrilled by one `tmpBuffer` and one `buffer`. I know this isn't your code so it isn't very clear what they stand for :-(. Btw -- could we do something *without* one of these buffers directly writing to the output? It would save us nontrivial stack space ...

Reply:
Probably not. If necessary, I might try to find other implementation.